roles/nova-common/templates/etc/apache2/sites-available/nova_placement_api.conf

```diff
@@ -0,0 +1,15 @@
+Listen 9775
+WSGIPythonHome {{ nova.nova_lib_dir }}
+WSGIPythonPath /etc/nova
+
+<VirtualHost *:8080>
```

Comment:
It ships with nova as a uswgi script. https://github.com/openstack/nova/blob/stable/newton/setup.cfg#L72-L73
https://docs.openstack.org/developer/nova/placement.html#steps